File: 3209.py
qtd_tomadas = 0

for i in range(int(input())):
  entrada = list(map(int, input().split()))
  qtd_filtros = entrada[0]
  filtros = entrada[1:]
  
  for j in range(qtd_filtros):
    if j == qtd_filtros-1:
      qtd_tomadas += filtros[j]
    else:
      qtd_tomadas += filtros[j]-1

  print(qtd_tomadas)
  qtd_tomadas = 0

File: 3209.c
#include <stdio.h>

int main() {
  int n, k, qtd_tomadas;

  scanf("%d", &n);

  for (int i = 0; i < n; i++) {
    qtd_tomadas = 0;
    scanf("%d", &k);

    for (int j = 0; j < k; j++) {
      int f;
      scanf("%d", &f);

      if(j == k-1) {
        qtd_tomadas += f;
      } else {
        qtd_tomadas += f-1;
      }
    }

    printf("%d\n", qtd_tomadas);
  }

  return 0;
}